fuzzamoto-ir/src/lib.rs

```diff
@@ -335,6 +344,40 @@ pub enum ProbeResult {
         /// The reason for why it failed to decode
         reason: String,
     },
+    RecentBlockes {
+        result: Vec<RecentBlock>,
+    },
 }
 
 pub type ProbeResults = Vec<ProbeResult>;
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub struct RecentBlock {
+    /// height of this block
+    pub height: u64,
+    /// The hash of a recent block
+    pub hash: [u8; 32],
+    /// Variable index of this block if it is defined in the testcase
+    pub defining_block: Option<(usize, usize)>,
```

Comment:
So currently the hash is mainly used if it is a block from the snapshot state and otherwise `defining_block` is used, right?

Could we instead make this always hold the header variables and required indices? Then we wouldn't need this distinction because `LoadHeader` also just returns a header variable.

Reply:
yes that makes more sense.
I'll do it

Reply:
One alternative could be just hardcode a 200th header into the Generator.
then we design the generator to sometimes generate a instruction that build block onto the 200th block, whose header hash we know.
This would work
But imo this is not ideal because it is dependent on the snapshotted state and the generator has to know beforehand which block is on the tip